lib/sentry/plug_capture.ex

```diff
@@ -66,4 +117,38 @@ defmodule Sentry.PlugCapture do
       end
     end
   end
+
+  @doc false
+  def __capture_exception__(exception, stacktrace, scrubber) do
+    # We can't pattern match here, because we're not guaranteed to have
+    # Phoenix available.
+    exception =
+      if is_struct(exception, Phoenix.ActionClauseError) do
```

Comment:
isn't doing this just for `ActionClauseError` too limited?
like it solves the issue but similar leakage could happen in other exceptions too right?